components/pedigree/resources/src/main/resources/pedigree/view/abstractHoverbox.js

```diff
@@ -267,6 +270,25 @@ define([
                 this.generateHandles();
         },
 
+        setDoNotHideHoverbox: function(raphaelElement) {
+            // raphaelElement is either a set which has .forEach() defined (but has no .node),
+            // or a single element which has .node but not .forEach()
+            if (raphaelElement.node) {
+                raphaelElement.node.setAttribute('class', 'donothidehoverbox ' + raphaelElement.node.getAttribute('class'));
+                // a hack for raphael <tspan> complication when using SVG (on most modern browsers)
+                if (raphaelElement.node.children &&
+                    raphaelElement.node.children.length > 0 &&
+                    raphaelElement.node.children[0].nodeName == "tspan") {
+                    raphaelElement.node.children[0].setAttribute("class","donothidehoverbox");
+                }
+            } else {
+                var _this = this;
+                raphaelElement.forEach(function(element) {
+                    _this.setDoNotHideHoverbox(element);
+                });
+            }
+        },
```

Comment:
re: screenshot: right, I see a problem (does not affect functionality, still repetitions and null-s better be fixed)

re: a lot of elements with this class name: well, this is what it has to be, is there an inherent problem with this? We just have a LOT of elements in each hoverbox (e.g. "alive & well" is composed of like 20 pieces, then 4 handles x a few pieces each, etc, etc), and a lot of hoverboxes.